docs/api/table/inputs.md

```diff
@@ -102,6 +102,14 @@ to select a particular row based on a criteria. Example:
 (row, column, value) => { return value !== 'Ethel Price'; }
 ```
 
+## `displayCheck`
+A function you can use to check whether you want
+to show the checkbox for a particular row based on a criteria. Example:
+
+```
+(row) => { return row.name !== 'Ethel Price'; }
+```
+
```

Comment:
@ShinDarth have you thought about adding column and cell value data in the function similar to selectCheck?
`(row, column, value) => { return value !== 'Ethel Price'; }`

Reply:
@wizarrc can you show me an example about how that's implemented?

Reply:
done